test_scripts/Smoke/API/018_SendLocation_PositiveCase.lua

```diff
@@ -0,0 +1,98 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: SendLocation
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [SendLocation] SUCCESS: getting SUCCESS:Navigation.SendLocation()
+--
+-- Description:
+-- Mobile application sends valid SendLocation request and gets Navigation.SendLocation "SUCCESS" response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests SendLocation with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if Navigation interface is available on HMI
+-- SDL checks if SendLocation is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the Navigation part of request with allowed parameters to HMI
+-- SDL receives Navigation part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  longitudeDegrees = 1.1,
+  latitudeDegrees = 1.1,
+  locationName = "location Name",
+  locationDescription = "location Description",
+  addressLines = {
+    "line1",
+    "line2",
+  },
+  phoneNumber = "phone Number",
+  locationImage ={
+    value = "icon.png",
+    imageType = "DYNAMIC"
+  }
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFuncRPC(tbl)
+  local SLgroup = {
+    rpcs = {
+      SendLocation = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" }
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = SLgroup
```

Comment:
@GetmanetsIrina I would propose to use "point" notation for constant string keys in tables and "braces" notation for variable string keys and other types keys.
Examples:
```
tbl.policy_table.functional_groupings.NewTestCaseGroup = SLgroup
```
but
```
local groupName = "NewTestCaseGroup"
tbl.policy_table.functional_groupings[groupName ] = SLgroup
```

Reply:
@aderiabin , Done in 87403972486a44f8eb7fcf65dcbe04200182c50f